save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/ContestController.kt

```diff
@@ -408,17 +403,24 @@ internal class ContestController(
         }
         .map { (organization, contests) ->
             contestsRequest.map { contestRequest ->
-                contestService.updateContest(
-                    contestRequest.toContest(organization).apply { id = contests.single { name == it.get().name }.get().id }
-                )
+                contests.single {
+                    contestRequest.name == it.get().name && contestRequest.organizationName == it.get().organization.name
+                }
+                    .get()
```

Comment:
suggestion to change to `singleOrNull().orNotFound` to have an error in response

Reply:
But what if we have a batch of requests?
Let's say the first one crashes. What will happen with others?

Reply:
Some of requests could be applied while `i`-th and the later won't

Reply:
`single {}` returns Optional, `.get()` throws exception `NoSuchElementException`. I suggest to throw something meaningful

Reply:
Makes sense, will do it

Reply:
Fixed